docs/post-v1experimental-determineversion.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: POST /v1experimental/determineversion
+permalink: /post-v1experimental-determineversion/
+parent: API
+nav_order: 2
+---
+# POST /v1experimental/determineversion
+
+Given the source code of C/C++ libraries, 
+this endpoint attempts to find the closest library and version.
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
```

Comment:
```suggestion

## Experimental endpoint

This is a new, experimental API endpoint.  The purpose of the endpoint is to help determine the version of a given C/C++ library. It is difficult to know the correct version of C/C++ projects because there is not a centralized package manager within the ecosystem. This API endpoint helps bridge that gap. Once you have the likely version, you can use [POST v1/query](post-v1-query.md) or [POST v1/queryset](post-v1-queryset.md) to search for vulnerabilities. 

As this is an experimental feature, we would love to hear about your experience using it. If you give this a try, please consider [opening an issue](https://github.com/google/osv.dev/issues/new) and letting us know about any pain points or highlights. 
```

@another-rex I think we need something like this paragraph on this page. We want to be clear that it is experimental, explain a little about why it is needed, and solicit feedback. I am open to changing up the wording, but I think it is important to provide some context.

Reply:
Thanks! Agreed, this is important context.